src/bindings/python/tests/test_ngraph/test_basic.py

```diff
@@ -327,6 +327,29 @@ def test_set_argument():
     assert np.allclose(data1 + data2, output)
 
 
+def test_clone_model():
+    # Create an original model
+    shape = [2, 2]
+    parameter_a = ops.parameter(shape, dtype=np.float32, name="A")
+    parameter_b = ops.parameter(shape, dtype=np.float32, name="B")
+    model_original = ov.Model(parameter_a + parameter_b, [parameter_a, parameter_b])
+
+    # Make copies of it
+    model_copy1 = ov.utils.clone_model(model_original)
+    model_copy2 = model_original.clone_model()
+
+    # Make changes to the cloned models' inputs
+    model_copy1.reshape({"A": [3, 3], "B": [3, 3]})
+    model_copy2.reshape({"A": [3, 3], "B": [3, 3]})
+
+    original_model_shapes = [single_input.get_shape() for single_input in model_original.inputs]
+    model_copy1_shapes = [single_input.get_shape() for single_input in model_copy1.inputs]
+    model_copy2_shapes = [single_input.get_shape() for single_input in model_copy2.inputs]
+
+    assert original_model_shapes != model_copy1_shapes
+    assert original_model_shapes != model_copy2_shapes
+
```

Comment:
add one more assert as `model_copy1_shapes` and `model_copy2_shapes`  are the same